src/greek_language_model.js

```diff
@@ -218,7 +218,7 @@ export default class GreekLanguageModel extends LanguageModel {
       pronounClassRequired: false
     }
     if (inflection.hasOwnProperty(Feature.types.part)) {
-      if (inflection[Feature.types.part].value === Constants.POFS_PRONOUN) {
+      if (inflection[Feature.types.part].value === Constants.POFS_PRONOUN || inflection[Feature.types.part].value === Constants.POFS_NUMERAL || inflection[Feature.types.part].value === Constants.POFS_ARTICLE) {
```

Comment:
I think it probably be simpler to have it as:
`if ([Constants.POFS_PRONOUN, Constants.POFS_NUMERAL, Constants.POFS_ARTICLE].includes(inflection[Feature.types.part].value))`
What do you think?